config/generateRunFile.coffee

```diff
@@ -457,105 +457,61 @@ generateDev = (KONFIG, options) ->
       command -v kubectl          >/dev/null 2>&1 || { echo >&2 "I require kubectl. To install kubectl: \
               (curl -L -O https://storage.googleapis.com/kubernetes-release/release/v1.6.4/bin/$(uname | awk '{print tolower($0)}')/amd64/kubectl && chmod +x kubectl && mv kubectl /usr/local/bin/)"; exit 1;}
 
-      # for local development, minikube must be deleted and recreated if a disk resize is needed
       export CHANGE_MINIKUBE_NONE_USER=true
       sudo -E minikube start --vm-driver=none
 
-      sleep 60
+      sleep 90
 
       export NAMESPACE_DIR="${KONFIG_PROJECTROOT}/deployment/kubernetes/namespace.yaml"
-      export BACKEND_DIR="${KONFIG_PROJECTROOT}/deployment/kubernetes/backend-pod/containers.yaml"
+      export EXT_SERVICES_DIR="${KONFIG_PROJECTROOT}/deployment/kubernetes/external-services/ -R"
       export FRONTEND_DIR="${KONFIG_PROJECTROOT}/deployment/kubernetes/frontend-pod/client-containers.yaml"
+      export BACKEND_DIR="${KONFIG_PROJECTROOT}/deployment/kubernetes/backend-pod/"
+      export BUILD_DIR="${KONFIG_PROJECTROOT}/deployment/kubernetes/build-pod.yaml"
 
-      envsubst < deployment/kubernetes/backend-pod/containers.yaml.template > deployment/kubernetes/backend-pod/containers.yaml
-      envsubst < deployment/kubernetes/frontend-pod/client-containers.yaml.template > deployment/kubernetes/frontend-pod/client-containers.yaml
-
-      cp $BACKEND_DIR ${KONFIG_PROJECTROOT}/deployment/generated_files
+      cp $BACKEND_DIR ${KONFIG_PROJECTROOT}/deployment/generated_files -r
```

Comment:
if this `-r` meant for recursive, please move it next to the command itself with its corresponding capital case.